### DISCUSSION TITLE
What is license of the SLEAP?

### DISCUSSION THREAD

**Discussion by doubletaro** [talmolab/sleap#756] [Help!] [answered]
Is it possible for commercial use?

**Comment by Banboo-glich** [comment]
No problem.  SLEAP is BSD License.  It allows proprietary use and allows the software released under the license to be incorporated into proprietary products. 

Cf
https://github.com/talmolab/sleap/blob/develop/LICENSE
https://en.wikipedia.org/wiki/BSD_licenses#:~:text=Commercial%20license%20compatibility,-The%20FreeBSD%20project&text=The%20BSD%20License%20allows%20proprietary,usual%20commercial%20usages%20under%20them.

**Comment by talmo** [comment, marked as answer]
Hi, SLEAP uses a [modified BSD-3 license](https://github.com/talmolab/sleap/blob/develop/LICENSE) that permits academic and research use only. That said, depending on how you're planning to use SLEAP for commercial use, you may be ok anyway. In particular, if you don't plan to **redistribute** SLEAP with your application, you're probably fine to use it.

If you're not sure, feel free to reach out to `talmo@salk.edu` with some info on your intended use and we can sort it out.

Cheers,

Talmo

**Comment by doubletaro** [comment]
@Banboo-glich 
Thank you for your reply.
Somehow, I understood that it is okay to use it for commercial purposes.

@talmo 
I will check SLEAP in various ways. 
And if I really need commercial use, I'll contact you just in case!
